packages/formation/__tests__/js/additional-info/additional-info.test.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-unused-expressions */
+import { expect } from 'chai';
+import additionalInfoWidget from '../../../js/additional-info';
+
+function qS(rootNode, selector) {
+  return rootNode.querySelector(selector);
+}
+
+describe('static page additional information widget', () => {
+  const basicTemplate = `<div class="form-expanding-group additional-info-container">
+    <span id="tooltip-1">
+      <div class="additional-info-content">
+        <ul>
+          <li>info A</li>
+          <li>info B</li>
+          <li>info C</li>
+          <li>info D</li>
+        </ul>
+      </div>
+    </span>
+  </div>`;
+
+  afterEach(() => {
+    document.getElementsByTagName('html')[0].innerHTML = '';
```

Comment:
Couldn't find something in jest that acted like mocha's global `afterEach` - jest setups are called before each test _file_ is ran.